app/javascript/src/components/Dashboard/Contacts/Table/Profile.jsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+import { Typography, Avatar } from "neetoui";
+import { useTranslation } from "react-i18next";
+
+const Profile = ({ name }) => {
+  const { t } = useTranslation();
+
+  return (
+    <div className="flex items-center space-x-3">
+      <Avatar
+        user={{
+          name,
+          imageUrl: name === "Jacob Jones" && "https://i.pravatar.cc/300",
+        }}
+      />
+      <div>
```

Comment:
Use a span here instead of a div.

Reply:
Fixed in [a6a0152](https://github.com/vysakh-poulose/neeto-ui-challenge-by-vysakh-poulose/pull/16/commits/a6a015270bc9f89e67d1b63dd49b9739dd03cab9)